examples/utils.py

```diff
@@ -1,9 +1,16 @@
+import os
 import time
 from collections.abc import Callable, Iterable
 from typing import Any
 
+CI_MODE = os.getenv("CI_MODE", default=False)
+
 
 def benchmark(func: Callable, args: Iterable[Any], info: str, iters: int):
+    if CI_MODE:
```

Comment:
We should have a warmup here.

Reply:
Warmup, or a precompilation stage, happens in each example file before `benchmark` is called. It isn't placed here as we also want the result of that call for `assert_allclose` statements.

Reply:
We could return from this function which is the result of the warmup call.

Reply:
Done!